src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -47,7 +47,6 @@
 
         .bcs-ActivityItem {
             position: relative;
-            padding: 20px $container-padding;
```

Comment:
Do we need to null out any browser-applied styles from the `li` element in this component?

Reply:
Doesn't seem like it but I'll double check the browsers